src/crypto/index.ts

```diff
@@ -1888,7 +1888,11 @@ export class Crypto extends TypedEventEmitter<CryptoEvent, CryptoEventHandlerMap
     /**
      * Implementation of {@link CryptoBackend#importBackedUpRoomKeys}.
      */
-    public importBackedUpRoomKeys(keys: IMegolmSessionData[], opts: ImportRoomKeysOpts = {}): Promise<void> {
+    public importBackedUpRoomKeys(
```

Comment:
Does this change a public interface and do we need to do something if so?

Reply:
no, CryptoBackend is an internal interface.

Reply:
(and legacy `Crypto` is documented as being an internal interface, even though we weren't very good at following that.)